package.json

```diff
@@ -93,65 +100,28 @@
 		"npm": "^7.0.0"
 	},
 	"devDependencies": {
-		"@babel/core": "^7.17.12",
-		"@babel/eslint-parser": "^7.18.2",
 		"@cypress/browserify-preprocessor": "^3.0.2",
 		"@cypress/webpack-preprocessor": "^5.12.0",
-		"@nextcloud/axios": "^1.10.0",
 		"@nextcloud/babel-config": "^1.0.0",
 		"@nextcloud/browserslist-config": "^2.2.0",
-		"@nextcloud/dialogs": "^3.1.2",
 		"@nextcloud/eslint-config": "^8.0.0",
-		"@nextcloud/eslint-plugin": "^2.0.0",
-		"@nextcloud/event-bus": "^2.1.1",
-		"@nextcloud/initial-state": "^1.2.1",
-		"@nextcloud/l10n": "^1.6.0",
-		"@nextcloud/moment": "^1.2.1",
 		"@nextcloud/stylelint-config": "^2.1.2",
-		"@nextcloud/vue": "^5.3.1",
 		"@nextcloud/webpack-vue-config": "^5.1.0",
 		"@vue/test-utils": "^1.3.0",
 		"@vue/vue2-jest": "^27.0.0",
-		"babel-loader": "^8.2.5",
-		"core-js": "^3.22.7",
-		"css-loader": "^6.7.1",
 		"cypress": "^10.1.0",
 		"cypress-file-upload": "^5.0.8",
 		"debounce": "^1.2.1",
 		"escape-html": "^1.0.3",
-		"eslint": "^8.18.0",
```

Comment:
This is a peer dependency of `@nextcloud/eslint`. My understanding is that peerDependencies of dependencies should be added as devDependencies to ensure they are installed and keep track of their versions even though npm will auto install peer dependencies nowadays.

Reply:
> This is a peer dependency of `@nextcloud/eslint`. My understanding is that peerDependencies of dependencies should be added as devDependencies to ensure they are installed and keep track of their versions even though npm will auto install peer dependencies nowadays.

It's the other way, we made sure each config dependency handle their required versions themselves.
All the other apps are doing it that way, it actually helped us a lot to reduce noise and ensure further compatibilities